examples/run_case_with_fork.py

```diff
@@ -1,31 +1,39 @@
 import flow360 as fl
 from flow360.examples import OM6wing
+from flow360 import log
 
 OM6wing.get_files()
+fl.Env.dev.active()
+log.set_logging_level('DEBUG')
 
 # submit mesh
 volume_mesh = fl.VolumeMesh.from_file(OM6wing.mesh_filename, name="OM6wing-mesh", tags=["tag"])
 volume_mesh = volume_mesh.submit()
 print(volume_mesh)
 
+volume_mesh2 = fl.VolumeMesh.from_file(OM6wing.mesh_filename, name="OM6wing-mesh2", tags=["tag"])
+volume_mesh2 = volume_mesh2.submit()
+print(volume_mesh2)
+
 
 # submit case using json file
 params = fl.Flow360Params(OM6wing.case_json)
-case = fl.Case.create("OM6wing", params, volume_mesh.id)
+case = fl.Case.create("OM6wing", params, volume_mesh.id, solver_version='release-23.3.2.0')
 case = case.submit()
 
 # fork a case
-case_fork_1 = case.fork()
+case_fork_1 = case.fork(interpolate_on_mesh_id=volume_mesh2.id)
```

Comment:
There are two ways of designing this feature:
1. fork to a new mesh id (which is what is done here)
2. create from a parent case id

Should we support both?